src/main/java/ladder/domain/Row.java

```diff
@@ -0,0 +1,43 @@
+package ladder.domain;
+
+import ladder.exception.LineDuplicateException;
+
+import java.util.List;
+
+public class Row {
+
+    private final List<Boolean> cols;
+
+    public Row(List<Boolean> cols) {
+        this.cols = cols;
+        validate();
+    }
+
+    private void validate() {
+        for (int i = 0; i < cols.size() - 1; i++) {
+            validateDuplicate(i);
+        }
+    }
+
+    private void validateDuplicate(int index) {
+        Boolean col1 = cols.get(index);
```

Comment:
```suggestion
        Boolean col1 = this.find(index);
```

`cols.get`호출 코드가 많이 보이네요, 하나의 메소드로 도출해내도 좋을것 같아요.